src/lib/Libifl/pbs_loadconf.c

```diff
@@ -277,7 +277,7 @@ parse_config_line(FILE *fp, char **key, char **val)
  * @retval 0 Failure
  */
 int
-pbs_loadconf(int reload)
+__pbs_loadconf(int reload)
```

Comment:
Why are you using the double underscore prefix? You've defined a macro for pbs_loadconf().